tflitehub/efficientnet_lite0_int8_test.py

```diff
@@ -0,0 +1,31 @@
+# RUN: %PYTHON %s
+
+import absl.testing
+import imagenet_test_data
+import numpy
+import test_util
+
+# Source https://tfhub.dev/tensorflow/lite-model/efficientnet/lite0/int8/2
+model_path = "https://storage.googleapis.com/iree-model-artifacts/efficientnet_lite0_int8_2.tflite"
+
+class EfficientnetLite0Int8Test(test_util.TFLiteModelTest):
+  def __init__(self, *args, **kwargs):
+    super(EfficientnetLite0Int8Test, self).__init__(model_path, *args, **kwargs)
+
+  def compare_results(self, iree_results, tflite_results, details):
+    super(EfficientnetLite0Int8Test, self).compare_results(iree_results, tflite_results, details)
+    # Dequantize outputs.
+    zero_point = details[0]['quantization_parameters']['zero_points'][0]
+    scale = details[0]['quantization_parameters']['scales'][0]
+    dequantized_iree_results = (iree_results - zero_point) * scale
+    dequantized_tflite_results = (tflite_results - zero_point) * scale
+    self.assertTrue(numpy.isclose(dequantized_iree_results, dequantized_tflite_results, atol=5e-3).all())
```

Comment:
OOC how are these bounds determined?

Reply:
They are metadata that's part of the tensor. The values are calculated during training - a running count of the min/max values are kept for each tensor and during the conversion step from Tensorflow to TFLite, these min/max values are used to convert the tensor values from FP32 to UINT8 (or INT8). Quantization parameters 'zero_point' and 'scale' are kept around in order to convert the quantized values back to float. For details: https://arxiv.org/abs/1712.05877 (or ask Benoit Jacob who authored the paper :)